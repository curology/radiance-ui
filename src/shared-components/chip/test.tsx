import React from 'react';
import { render } from 'src/tests/testingLibraryHelpers';

import { Chip } from './index';

describe('<Chip />', () => {
  describe('UI snapshots', () => {
    it('renders the correct css and text', () => {
      const { container } = render(
        <Chip
          status="success"
          text="Success"
        />,
      );

      expect(container.firstElementChild).toMatchSnapshot();
    });
  });
});
