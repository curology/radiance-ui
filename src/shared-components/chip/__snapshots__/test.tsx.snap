// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> UI snapshots renders the correct css and text 1`] = `
.emotion-0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 4px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 1.5rem;
  -webkit-box-flex-flow: row nowrap;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 0.5rem;
  background-color: #2B6E33;
}

.emotion-0 .emotion-3 {
  color: #FFFFFF;
}

.emotion-2 {
  color: #524D6E;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  -webkit-transform: translateY(1px);
  -moz-transform: translateY(1px);
  -ms-transform: translateY(1px);
  transform: translateY(1px);
}

<div
  class="emotion-0 emotion-1"
>
  <span
    class="emotion-2 emotion-3"
  >
    Success
  </span>
</div>
`;
